lago/utils.py

```diff
@@ -421,6 +421,10 @@ def deepcopy(original_obj):
     """
     if isinstance(original_obj, list):
         return list(deepcopy(item) for item in original_obj)
+    elif isinstance(original_obj, collections.OrderedDict):
+        return collections.OrderedDict(
+            (key, deepcopy(val)) for key, val in original_obj.items()
+        )
```

Comment:
Hmm, that was a good explanation on why any deepcopy is needed, not on why use our own deepcopy :/, my bad.

An example of that issue:
```
In [36]: import yaml
In [37]: myyaml='''
something: &common_stuff
  key: value
  key2: value2
  key3:
    - one
    - two

one:
  <<: *common_stuff
two:
  <<: *common_stuff

'''
In [38]: res = yaml.load(myyaml)
In [39]: res
Out[39]: 
{'one': {'key': 'value', 'key2': 'value2', 'key3': ['one', 'two']},
 'something': {'key': 'value', 'key2': 'value2', 'key3': ['one', 'two']},
 'two': {'key': 'value', 'key2': 'value2', 'key3': ['one', 'two']}}
In [40]: res['one']
Out[40]: {'key': 'value', 'key2': 'value2', 'key3': ['one', 'two']}
In [41]: res['one'] is res['two']
Out[41]: False
In [42]: res['one']['key3'] is res['two']['key3']
Out[42]: True
```
I don't see any issue overall, I'll try replacing it with the stdlib one and do some tests, see if it breaks anything.